third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -718,6 +718,22 @@ func resourceContainerCluster() *schema.Resource {
 				},
 			},
 
+			"workload_identity_config": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"identity_namespace": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
Is it meaningful to set the block above without setting this value? I'd imagine we can make this `Required`.